.github/workflows/versioning.yml

```
@@ -25,21 +27,36 @@
+          git checkout main
+          git pull https://x-access-token:${GITHUB_TOKEN}@github.com/${{ github.repository }}.git main
+          git merge --no-ff $TEMP_BRANCH -m "Automated Version Bump [skip-versioning]"
+          git push https://x-access-token:${GITHUB_TOKEN}@github.com/${{ github.repository }}.git main
 
-      - name: Conditionally Update Stable Tag
-        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.update_stable == 'true' }}
+      - name: Delete Temporary Branch
+        if: always()
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:33: Double quote to prevent globbing and word splitting

(shellcheck)